plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -285,13 +286,20 @@ protected function appendLiveParams(LiveStreamEntry $entry, WowzaMediaServerNode
 		$keyFrameInterval->addChild('FollowSource', 'true');
 		$keyFrameInterval->addChild('Interval', 60);
 
-		$skipFrameCountPos = strpos($liveParams->getConversionEnginesExtraParams(), 'SkipFrameCount');
-		if ($skipFrameCountPos !== false)
-		{
-			$skipFrameCount = $video->addChild('SkipFrameCount');
-			preg_match('/SkipFrameCount=(\d+)/', $liveParams->getConversionEnginesExtraParams(), $skipFrameValue, $skipFrameCountPos);
-			$skipFrameCount->addChild('Value', (int)$skipFrameValue[1]);
-		}
+        if ($conversionExtraParam !== null && $conversionExtraParam->skipFrameCount !== null)
+        {
+            $skipFrameCount = $video->addChild('SkipFrameCount');
+            $skipFrameCount->addChild('Value', (int)$conversionExtraParam->skipFrameCount->value);
```

Comment:
Why not simply use $conversionExtraParam->skipFrameCount and json that looks like {"skipFrameCount":8} 

Also don't need to casting to int